nyaa/templates/layout.html

```diff
@@ -90,6 +90,7 @@
 						{% endif %}
 						{% if g.user.is_moderator %}
 							<li><a href="{{ url_for('view_reports') }}">Reports</a> </li>
+                            <li><a href="{{ url_for('view_adminlog') }}">Log</a> </li>
```

Comment:
1. Spaces should be tabs
2. Maybe add a drop-down until such point when everything admin-related is in a dedicated admin section?
Not sure about the title for it, though. Simply "Admin"?

What I mean:
![image](https://user-images.githubusercontent.com/10238474/27156155-8d2f928a-5164-11e7-8504-dc06291be35b.png)

**Code**: (replaces lines 91-94 , mostly copied from the "About" nav item)
```jinja
{% if g.user.is_moderator %}
<li class="dropdown">
	<a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
		Admin
		<span class="caret"></span>
	</a>
	<ul class="dropdown-menu">
		<li {% if request.path == url_for('view_reports') %}class="active"{% endif %}><a href="{{ url_for('view_reports') }}">Reports</a></li>
		<li {% if request.path == url_for('view_adminlog') %}class="active"{% endif %}><a href="{{ url_for('view_adminlog') }}">Log</a></li>
	</ul>
</li>
{% endif %}
```